`auto_upgrade` doesn't work anymore

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I have OhMyPosh installed on Powershell (inside Windows Terminal).
Today I noticed that OhMyPosh no longer checks for new versions at launch, e.g. the very latest 21.20.1.

I have a "custom" theme, that is `jandedobbeleer.omp.json` with the addition of these two settings:

```json
  "auto_upgrade": true,
  "disable_notice": true
```
So, as [per](https://github.com/JanDeDobbeleer/oh-my-posh/issues/5166#issuecomment-2187987606) I have copied `jandedobbeleer.omp.json` renaming it e.g. `my_custom_theme.omp.json`, added the two above settings, and put it in %appdata%, i.e. `C:\Users\<User>\AppData\Roaming`.

I also made a Windows Terminal profile with this entry:

```powershell
oh-my-posh init pwsh --config ~/my_custom_theme.omp.json | Invoke-Expression
```

### Theme

The entire `jandedobbeleer.omp.json` config file with the addition of two settings: 

&nbsp;
The only settings I modified which are gone after closing the terminal are:
```
  "auto_upgrade": true,
  "disable_notice": true
```

that is:

<details>
<summary><code>my_custom_theme.omp.json</code> (click to expand):</summary>

```json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "background": "#c386f1",
          "foreground": "#ffffff",
          "leading_diamond": "\ue0b6",
          "style": "diamond",
          "template": " {{ .UserName }} ",
          "trailing_diamond": "\ue0b0",
          "type": "session"
        },
        {
          "background": "#ff479c",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "folder_separator_icon": " \ue0b1 ",
            "home_icon": "~",
            "style": "folder"
          },
          "style": "powerline",
          "template": " \uea83  {{ .Path }} ",
          "type": "path"
        },
        {
          "background": "#fffb38",
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}#FF9248{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}#ff4500{{ end }}",
            "{{ if gt .Ahead 0 }}#B388FF{{ end }}",
            "{{ if gt .Behind 0 }}#B388FF{{ end }}"
          ],
          "foreground": "#193549",
          "leading_diamond": "\ue0b6",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "branch_max_length": 25,
            "fetch_stash_count": true,
            "fetch_status": true,
            "fetch_upstream_icon": true
          },
          "style": "powerline",
          "template": " {{ .UpstreamIcon }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }} \ueb4b {{ .StashCount }}{{ end }} ",
          "trailing_diamond": "\ue0b4",
          "type": "git"
        },
        {
          "background": "#6CA35E",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "fetch_version": true
          },
          "style": "powerline",
          "template": " \ue718 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} ",
          "type": "node"
        },
        {
          "background": "#8ED1F7",
          "foreground": "#111111",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "fetch_version": true
          },
          "style": "powerline",
          "template": " \ue626 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "type": "go"
        },
        {
          "background": "#4063D8",
          "foreground": "#111111",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "fetch_version": true
          },
          "style": "powerline",
          "template": " \ue624 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "type": "julia"
        },
        {
          "background": "#FFDE57",
          "foreground": "#111111",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "display_mode": "files",
            "fetch_virtual_env": false
          },
          "style": "powerline",
          "template": " \ue235 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "type": "python"
        },
        {
          "background": "#AE1401",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "display_mode": "files",
            "fetch_version": true
          },
          "style": "powerline",
          "template": " \ue791 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "type": "ruby"
        },
        {
          "background": "#FEAC19",
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "display_mode": "files",
            "fetch_version": false
          },
          "style": "powerline",
          "template": " \uf0e7{{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
          "type": "azfunc"
        },
        {
          "background_templates": [
            "{{if contains \"default\" .Profile}}#FFA400{{end}}",
            "{{if contains \"jan\" .Profile}}#f1184c{{end}}"
          ],
          "foreground": "#ffffff",
          "powerline_symbol": "\ue0b0",
          "properties": {
            "display_default": false
          },
          "style": "powerline",
          "template": " \ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} ",
          "type": "aws"
        },
        {
          "background": "#ffff66",
          "foreground": "#111111",
          "powerline_symbol": "\ue0b0",
          "style": "powerline",
          "template": " \uf0ad ",
          "type": "root"
        },
        {
          "background": "#83769c",
          "foreground": "#ffffff",
          "properties": {
            "always_enabled": true
          },
          "style": "plain",
          "template": "<transparent>\ue0b0</> \ueba2 {{ .FormattedMs }}\u2800",
          "type": "executiontime"
        },
        {
          "background": "#00897b",
          "background_templates": [
            "{{ if gt .Code 0 }}#e91e63{{ end }}"
          ],
          "foreground": "#ffffff",
          "properties": {
            "always_enabled": true
          },
          "style": "diamond",
          "template": "<parentBackground>\ue0b0</> \ue23a ",
          "trailing_diamond": "\ue0b4",
          "type": "status"
        }
      ],
      "type": "prompt"
    },
    {
      "segments": [
        {
          "background": "#0077c2",
          "foreground": "#ffffff",
          "style": "plain",
          "template": "<#0077c2,transparent>\ue0b6</> \uf489 {{ .Name }} <transparent,#0077c2>\ue0b2</>",
          "type": "shell"
        },
        {
          "background": "#1BD760",
          "foreground": "#111111",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "paused_icon": "\uf04c ",
            "playing_icon": "\uf04b "
          },
          "style": "powerline",
          "template": " \uf167 {{ .Icon }}{{ if ne .Status \"stopped\" }}{{ .Artist }} - {{ .Track }}{{ end }} ",
          "type": "ytm"
        },
        {
          "background": "#f36943",
          "background_templates": [
            "{{if eq \"Charging\" .State.String}}#40c4ff{{end}}",
            "{{if eq \"Discharging\" .State.String}}#ff5722{{end}}",
            "{{if eq \"Full\" .State.String}}#4caf50{{end}}"
          ],
          "foreground": "#ffffff",
          "invert_powerline": true,
          "powerline_symbol": "\ue0b2",
          "properties": {
            "charged_icon": "\ue22f ",
            "charging_icon": "\ue234 ",
            "discharging_icon": "\ue231 "
          },
          "style": "powerline",
          "template": " {{ if not .Error }}{{ .Icon }}{{ .Percentage }}{{ end }}{{ .Error }}\uf295 ",
          "type": "battery"
        },
        {
          "background": "#2e9599",
          "foreground": "#111111",
          "invert_powerline": true,
          "leading_diamond": "\ue0b2",
          "style": "diamond",
          "template": " {{ .CurrentDate | date .Format }} ",
          "trailing_diamond": "\ue0b4",
          "type": "time"
        }
      ],
      "type": "rprompt"
    }
  ],
  "console_title_template": "{{ .Shell }} in {{ .Folder }}",
  "final_space": true,
  "version": 2,
  "auto_upgrade": true,
  "disable_notice": true
}

```
</details>

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 21.19.0

Shell: pwsh (7.4.3)

Prompt:

 Kostas    ~   0ms⠀                                                                       pwsh  23:36:25 
Segments:

ConsoleTitle(true)                         -   1 ms
Session(true)                              -   0 ms
Path(true)                                 -   1 ms
Executiontime(true)                        -   0 ms
Status(true)                               -   0 ms
Shell(true)                                -   0 ms
Time(true)                                 -   0 ms

Run duration: 26.7836ms

Cache path: C:\Users\Kostas\AppData\Local\oh-my-posh

Config path: C:\Users\Kostas\my_custom_theme.omp.json

Logs:

[DEBUG] 23:36:25.889 os.go:Getenv:306 → C:\Users\Kostas\AppData\Local
[TRACE] 23:36:25.890 os.go:Getenv(LOCALAPPDATA) - 546.3µs
[TRACE] 23:36:25.890 os_windows.go:CachePath() - 546.3µs
[DEBUG] 23:36:25.890 os.go:Getenv:306 → C:\Users\Kostas\my_custom_theme.omp.json
[TRACE] 23:36:25.890 os.go:Getenv(POSH_THEME) - 0s
[DEBUG] 23:36:25.890 os.go:Shell:581 → no shell name provided in flags, trying to detect it
[DEBUG] 23:36:25.898 os.go:Shell:589 → process name: pwsh.exe
[TRACE] 23:36:25.898 os.go:Shell() - 7.7381ms
[TRACE] 23:36:25.898 os.go:resolveConfigPath() - 7.7381ms
[TRACE] 23:36:25.898 os.go:Init() - 8.8313ms
[TRACE] 23:36:25.898 os.go:Flags() - 0s
[TRACE] 23:36:25.899 load.go:loadConfig() - 637.2µs
[TRACE] 23:36:25.899 os.go:Flags() - 0s
[DEBUG] 23:36:25.899 debug.go:44 → Terminal shell: %s shell
[DEBUG] 23:36:25.899 debug.go:44 → Terminal program: %s Windows Terminal
[DEBUG] 23:36:25.899 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.899 os.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 23:36:25.899 os_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[TRACE] 23:36:25.899 os.go:GOOS() - 0s
[DEBUG] 23:36:25.899 os_windows.go:WindowsRegistryKeyValue:217 → ColorizationColor(DWORD): 0xC40078D4
[TRACE] 23:36:25.899 os.go:Shell() - 0s
[DEBUG] 23:36:25.899 os.go:Getenv:306 → 7.4.3
[TRACE] 23:36:25.899 os.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 23:36:25.899 debug.go:PrintDebug:23 → Segment: Title
[DEBUG] 23:36:25.899 text.go:Render:73 → Rendering template: {{ .Shell }} in {{ .Folder }}
[TRACE] 23:36:25.900 os_windows.go:Root() - 0s
[TRACE] 23:36:25.900 os.go:Shell() - 0s
[TRACE] 23:36:25.900 os.go:StatusCodes() - 0s
[TRACE] 23:36:25.900 os_windows.go:IsWsl() - 0s
[DEBUG] 23:36:25.900 os.go:TemplateCache:779 → environment: [ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\Kostas\AppData\Roaming chocolateyinstall=C:\Users\Kostas\AppData\Local\UniGetUI\Chocolatey CLINK_DIR=C:\Program Files (x86)\clink CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=KOSTAS-PC ComSpec=C:\WINDOWS\system32\cmd.exe CONDA_PROMPT_MODIFIER=False DriverData=C:\Windows\System32\Drivers\DriverData HOMEDRIVE=C: HOMEPATH=\Users\Kostas JD2_HOME=C:\Users\Kostas\AppData\Local\JDownloader 2.0 LOCALAPPDATA=C:\Users\Kostas\AppData\Local LOGONSERVER=\\KOSTAS-PC NUMBER_OF_PROCESSORS=8 OneDrive=C:\Users\Kostas\OneDrive OS=Windows_NT Path=C:\Program Files\PowerShell\7;C:\Program Files (x86)\VMware\VMware Workstation\bin\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Microsoft VS Code\bin;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\NVIDIA Corporation\NVIDIA App\NvDLISR;C:\Program Files\Meld\;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\Program Files\PowerShell\7\;C:\Program Files\GitHub CLI\;C:\Program Files\nodejs\;C:\Users\Kostas\AppData\Local\Programs\Python\Launcher\;C:\Users\Kostas\AppData\Local\UniGetUI\Chocolatey\bin;C:\Users\Kostas\AppData\Local\Programs\WingetUI\choco-cli\bin;C:\Users\Kostas\AppData\Local\Microsoft\WindowsApps;C:\Users\Kostas\AppData\Local\GitHubDesktop\bin;C:\Users\Kostas\.deno\bin;C:\Users\Kostas\AppData\Local\Microsoft\WindowsApps;C:\GitHub CLI\;C:\Users\Kostas\AppData\Local\Programs\oh-my-posh\bin;C:\Users\Kostas\.bun\bin;C:\Users\Kostas\AppData\Local\Microsoft\WinGet\Links;C:\Users\Kostas\AppData\Local\Programs\nu\bin\;C:\Users\Kostas\AppData\Roaming\npm PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL POSH_AZURE_ENABLED=False POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=1 POSH_GIT_ENABLED=False POSH_INSTALLER=manual POSH_PID=26484 POSH_SHELL_VERSION=7.4.3 POSH_THEME=C:\Users\Kostas\my_custom_theme.omp.json POSH_THEMES_PATH=C:\Users\Kostas\AppData\Local\Programs\oh-my-posh\themes POWERLINE_COMMAND=oh-my-posh POWERSHELL_DISTRIBUTION_CHANNEL=MSI:Windows 10 Pro PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 158 Stepping 12, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=9e0c ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\Kostas\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules PUBLIC=C:\Users\Public SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\Kostas\AppData\Local\Temp TMP=C:\Users\Kostas\AppData\Local\Temp USERDOMAIN=KOSTAS-PC USERDOMAIN_ROAMINGPROFILE=KOSTAS-PC USERNAME=Kostas USERPROFILE=C:\Users\Kostas windir=C:\WINDOWS WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=82ac19f5-57d2-4d76-954a-8b5825512603]
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 os.go:Pwd:336 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os_windows.go:IsWsl() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 os.go:User:474 → Kostas
[TRACE] 23:36:25.901 os.go:User() - 0s
[DEBUG] 23:36:25.901 os.go:Host:491 → Kostas-PC
[TRACE] 23:36:25.901 os.go:Host() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:Getenv(SHLVL) - 0s
[TRACE] 23:36:25.901 os.go:TemplateCache() - 747.5µs
[DEBUG] 23:36:25.901 os.go:Getenv:306 → 1
[TRACE] 23:36:25.901 os.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 23:36:25.901 os.go:Getenv:306 → 1
[TRACE] 23:36:25.901 os.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Shell() - 0s
[TRACE] 23:36:25.901 os.go:Shell() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 status.go:Init:53 → {{ .Code }}
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Session
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 python.go:Init:61 → files
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Status
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:StatusCodes() - 0s
[TRACE] 23:36:25.901 os.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 23:36:25.901 text.go:Render:73 → Rendering template: {{ .Code }}
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 23:36:25.901 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Python
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Aws
[DEBUG] 23:36:25.901 language.go:Enabled:109 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 23:36:25.901 language.go:Enabled:110 → folders: [.venv venv virtualenv venv-win pyenv-win]
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:117 → home_enabled: false
[DEBUG] 23:36:25.901 aws.go:Enabled:42 → display_default: false
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:Getenv(AWS_VAULT) - 0s
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Getenv(AWS_DEFAULT_PROFILE) - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:110 → folders: []
[TRACE] 23:36:25.901 os.go:Getenv(AWS_PROFILE) - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:117 → home_enabled: false
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[TRACE] 23:36:25.901 os.go:Getenv(AWS_REGION) - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os.go:Getenv:306 → NO DATA
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:Getenv(AWS_DEFAULT_REGION) - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Getenv(AWS_CONFIG_FILE) - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Julia
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 language.go:Enabled:109 → extensions: [*.jl]
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:110 → folders: []
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Go
[DEBUG] 23:36:25.901 language.go:Enabled:117 → home_enabled: false
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:109 → extensions: [*.go go.mod]
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 language.go:Enabled:110 → folders: []
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:117 → home_enabled: false
[DEBUG] 23:36:25.901 os.go:Getenv:306 → C:\Users\Kostas\AppData\Local
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Ruby
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 language.go:Enabled:109 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Root
[DEBUG] 23:36:25.901 language.go:Enabled:110 → folders: []
[DEBUG] 23:36:25.901 language.go:Enabled:117 → home_enabled: false
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 path.go:124 → display_cygpath: false
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os_windows.go:Root() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:PathSeparator() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Shell() - 0s
[TRACE] 23:36:25.901 os.go:Flags() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Azfunc
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 segment.go:SetEnabled:113 → Segment: Executiontime
[DEBUG] 23:36:25.901 language.go:Enabled:109 → extensions: [host.json local.settings.json function.json]
[DEBUG] 23:36:25.901 language.go:Enabled:110 → folders: []
[DEBUG] 23:36:25.901 language.go:Enabled:117 → home_enabled: false
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[DEBUG] 23:36:25.901 executiontime.go:Enabled:57 → always_enabled: true
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:ExecutionTime() - 0s
[DEBUG] 23:36:25.901 executiontime.go:Enabled:59 → threshold: 500.000000
[DEBUG] 23:36:25.901 executiontime.go:Enabled:63 → austin
[TRACE] 23:36:25.901 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.901 path.go:replaceMappedLocations:517 → mapped_locations_enabled: true
[DEBUG] 23:36:25.901 path.go:replaceMappedLocations:518 → 
[DEBUG] 23:36:25.901 path.go:replaceMappedLocations:519 → 
[TRACE] 23:36:25.901 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[DEBUG] 23:36:25.901 status.go:Enabled:42 → always_enabled: true
[TRACE] 23:36:25.901 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 path.go:replaceMappedLocations:520 → ~
[ERROR] 23:36:25.901 os.go:FileContent:444 → open C:\Users\Kostas/.aws/config: The system cannot find the file specified.
[DEBUG] 23:36:25.901 path.go:replaceMappedLocations:525 → mapped_locations: map[]
[TRACE] 23:36:25.901 os.go:FileContent(C:\Users\Kostas/.aws/config) - 503.2µs
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[TRACE] 23:36:25.901 os.go:GOOS() - 0s
[DEBUG] 23:36:25.901 path.go:makeFolderFormatMap:841 → NO DATA
[DEBUG] 23:36:25.901 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.901 os.go:Pwd() - 0s
[TRACE] 23:36:25.901 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.901 os.go:StackCount() - 0s
[DEBUG] 23:36:25.901 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 23:36:25.901 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 23:36:25.901 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 23:36:25.901 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 23:36:25.902 os_windows.go:DirIsWritable() - 0s
[TRACE] 23:36:25.902 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.911 os.go:CommandPath:533 → C:\Program Files\Git\cmd\git.exe
[TRACE] 23:36:25.911 os.go:CommandPath(git.exe) - 10.4285ms
[TRACE] 23:36:25.911 os.go:HasCommand(git.exe) - 10.4285ms
[TRACE] 23:36:25.911 os.go:Pwd() - 0s
[ERROR] 23:36:25.911 os.go:HasParentFilePath:687 → CreateFile .git: The system cannot find the file specified.
[TRACE] 23:36:25.911 os.go:HasParentFilePath(.git) - 0s
[DEBUG] 23:36:25.911 git.go:shouldDisplay:284 → fetch_bare_info: false
[DEBUG] 23:36:25.911 text.go:Render:73 → Rendering template:  {{ .UserName }}
[DEBUG] 23:36:25.911 text.go:Render:73 → Rendering template:    {{ .Path }}
[TRACE] 23:36:25.912 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.912 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.912 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: diamond
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: plain
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: {{ if gt .Code 0 }}#e91e63{{ end }}
[TRACE] 23:36:25.912 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: diamond
[DEBUG] 23:36:25.912 text.go:Render:73 → Rendering template: {{ if gt .Code 0 }}#e91e63{{ end }}
[TRACE] 23:36:25.913 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.913 os.go:Shell() - 0s
[TRACE] 23:36:25.913 os.go:Shell() - 0s
[TRACE] 23:36:25.913 os.go:Shell() - 0s
[TRACE] 23:36:25.913 os.go:Flags() - 0s
[TRACE] 23:36:25.913 os.go:Flags() - 0s
[TRACE] 23:36:25.913 os.go:Pwd() - 0s
[TRACE] 23:36:25.913 os.go:Pwd() - 0s
[DEBUG] 23:36:25.913 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.913 os.go:GOOS() - 0s
[DEBUG] 23:36:25.913 segment.go:SetEnabled:113 → Segment: Time
[DEBUG] 23:36:25.913 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.913 os.go:GOOS() - 0s
[DEBUG] 23:36:25.913 segment.go:SetEnabled:113 → Segment: Ytm
[DEBUG] 23:36:25.913 time.go:Enabled:29 → 15:04:05
[TRACE] 23:36:25.913 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.913 os.go:Flags() - 0s
[DEBUG] 23:36:25.913 ytm.go:setStatus:70 → http://127.0.0.1:9863
[TRACE] 23:36:25.913 os.go:Flags() - 0s
[TRACE] 23:36:25.913 os.go:Pwd() - 0s
[DEBUG] 23:36:25.913 ytm.go:setStatus:71 → api_url: 20
[DEBUG] 23:36:25.913 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.913 os.go:GOOS() - 0s
[DEBUG] 23:36:25.913 segment.go:SetEnabled:113 → Segment: Battery
[TRACE] 23:36:25.913 os.go:Pwd() - 0s
[TRACE] 23:36:25.913 os_windows.go:IsWsl() - 0s
[DEBUG] 23:36:25.913 os_windows.go:58 → C:\Users\Kostas
[TRACE] 23:36:25.913 os.go:GOOS() - 0s
[DEBUG] 23:36:25.913 segment.go:SetEnabled:113 → Segment: Shell
[DEBUG] 23:36:25.913 shell.go:Enabled:28 → mapped_shell_names: map[]
[TRACE] 23:36:25.913 os.go:Shell() - 0s
[TRACE] 23:36:25.913 os.go:Flags() - 0s
[TRACE] 23:36:25.913 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.913 os.go:HTTPRequest:637 ↓
    GET /query HTTP/1.1
    Host: 127.0.0.1:9863
    User-Agent: Go-http-client/1.1
    Accept-Encoding: gzip


[ERROR] 23:36:25.914 os.go:HTTPRequest:642 → Get "http://127.0.0.1:9863/query": dial tcp 127.0.0.1:9863: connectex: No connection could be made because the target machine actively refused it.
[TRACE] 23:36:25.914 os.go:HTTPRequest(http://127.0.0.1:9863/query) - 1.0538ms
[ERROR] 23:36:25.915 os_windows_nix.go:BatteryState:15 → no battery
[TRACE] 23:36:25.915 os_windows_nix.go:BatteryState() - 2.1644ms
[DEBUG] 23:36:25.915 text.go:Render:73 → Rendering template:  {{ .CurrentDate | date .Format }}
[DEBUG] 23:36:25.915 text.go:Render:73 → Rendering template: <#0077c2,transparent></>  {{ .Name }} <transparent,#0077c2></>
[TRACE] 23:36:25.915 os.go:TemplateCache() - 0s
[TRACE] 23:36:25.915 os.go:TemplateCache() - 0s
[DEBUG] 23:36:25.915 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.915 text.go:Render:73 → Rendering template: powerline
[DEBUG] 23:36:25.915 text.go:Render:73 → Rendering template: plain
[DEBUG] 23:36:25.915 text.go:Render:73 → Rendering template: diamond
[TRACE] 23:36:25.915 os.go:Shell() - 0s
[TRACE] 23:36:25.915 os.go:Flags() - 0s
[TRACE] 23:36:25.915 os.go:Shell() - 0s
[TRACE] 23:36:25.915 os.go:Shell() - 0s
[DEBUG] 23:36:25.916 os_windows.go:TerminalWidth:111 → terminal width: 120
[TRACE] 23:36:25.916 os_windows.go:TerminalWidth() - 548.9µs
[DEBUG] 23:36:25.916 os.go:Getenv:306 → C:\Users\Kostas\AppData\Local
[TRACE] 23:36:25.916 os.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 23:36:25.916 os_windows.go:CachePath() - 0s
[TRACE] 23:36:25.916 os.go:Flags() - 0s
 Kostas    ~   70ms⠀                                                                      pwsh 
```

--- a/src/cli/init.go
+++ b/src/cli/init.go
@@ -73,6 +73,7 @@ func runInit(shellName string) {
 	shell.Tooltips = len(cfg.Tooltips) > 0
 	shell.ShellIntegration = cfg.ShellIntegration
 	shell.PromptMark = shellName == shell.FISH && cfg.ITermFeatures != nil && cfg.ITermFeatures.Contains(terminal.PromptMark)
+	shell.AutoUpgrade = cfg.AutoUpgrade
 
 	for i, block := range cfg.Blocks {
 		// only fetch cursor position when relevant

